stefaniaPopescu26/assignment2/TreeTest.java

```diff
@@ -0,0 +1,51 @@
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class TreeTest {
+
+    public Tree<Integer> getTestTree1() {
```

Comment:
Try making namings of your methods more meaningful: for example (and this is just my suggestion here, feel free to go with your own idea):
getTestTree1 -> getTestTreeInts
getTestTree2 -> getTestTreeChars
testAdd1 -> testAddInts
testAdd2 - >testAddChars
testAdd3 -> testInOrederNull